build/paramfetch.go

```diff
@@ -54,6 +54,9 @@ func GetParams(storage bool, tests bool) error {
 	ft := &fetch{}
 
 	for name, info := range params {
+		if info.SectorSize != 1024 {
```

Comment:
protip: run tests with `TRUST_PARAMS=1`, which disables those checks

Reply:
it will still try to download them, no?

Reply:
Nope